### PR TITLE
Update set.xml

### PR DESCRIPTION
**Pull request by pilunte23**
fix clues

### CODE REVIEW

**Review by ChristopherS — CHANGES_REQUESTED** [review]

File: o8g/Scripts/actions.py [hunk 1]
```diff
@@ -1051,6 +1099,27 @@ def doMythosPhase(setPhaseVar = True):
         if card.Type == "Agenda" and card.controller == me and not isLocked(card) and card.isFaceUp:
             addDoom(card)
 
+def listBonded(deck):
+    mute()
+    listeBonded = []
+    for deckcard in deck:
+        for sca, bca , bco in zip(sourceCard, bondedCard, bondedCode):
+            if sca == deckcard.Name:
+                #Filter Dream Diary with wrong subtitle
```
[the remark this answers is not in the document]
Comment (pilunte23) [reply o8g/Scripts/actions.py:1108]: no , there are 4 "Dream Diary" version with same name but different subtitle , only "DreamDiary" with subtitle "Untranslated" have bonded card.

[the remark this answers is not in the document]
Comment (ChristopherS) [reply o8g/Scripts/actions.py:1108]: Ah ok, Then same as the comment below.

File: o8g/Scripts/actions.py [hunk 2]
```diff
@@ -1051,6 +1099,27 @@ def doMythosPhase(setPhaseVar = True):
...
+                if deckcard.Name == "Dream Diary" and deckcard.Subtitle != "Untranslated":
+                    continue              
+                listeBonded.extend(bco)
+    #Remove useless multiple bonded card and add only 1 bonded card
+    #Essence of the Dream
+    if "0481911c-f906-40ee-acef-d4ad39c2d767" in listeBonded:
```
[the remark this answers is not in the document]
Comment (pilunte23) [reply o8g/Scripts/actions.py:1114]: i try with this list

limitedOneBondedCode =[
    ["0481911c-f906-40ee-acef-d4ad39c2d767"],
    ["dc7600e3-a132-42b0-b96b-67ca0cee4aff"]
    ]

and this code 

    for idOne in limitedOneBondedCode:
        if idOne in listBonded:
            notify("{} find".format(idOne))
            listBonded = filter(lambda card: card != idOne, listBonded)
            listBonded.append(idOne) 

But i don't why i go in loop but my condition doesn't work.

[the remark this answers is not in the document]
Comment (ChristopherS) [reply o8g/Scripts/actions.py:1114]: `if idOne in listBonded:`

Your variable is called listeBonded

[the remark this answers is not in the document]
Comment (pilunte23) [reply o8g/Scripts/actions.py:1114]: no , it's not issue , i rename this variable in all my code
def listBonded(deck):
    mute()
    listBonded = []
    for deckcard in deck:
        for sca, bca , bco in zip(sourceCard, bondedCard, bondedCode):
            if sca == deckcard.Name:
                #Filter Dream Diary with wrong subtitle
                if deckcard.Name == "Dream Diary" and deckcard.Subtitle != "Untranslated":
                    continue              
                listBonded.extend(bco)
    #Remove useless multiple bonded card and add only 1 bonded card
    #Essence of the Dream
    #Pendant of the Queen
    for idOne in limitedOneBondedCode:
        test = idOne
        if test in listBonded:
            notify("{} find".format(test))
            listBonded = filter(lambda card: card != test, listBonded)
            listBonded.append(test)
    return listBonded

[the remark this answers is not in the document]
Comment (pilunte23) [reply o8g/Scripts/actions.py:1114]: Find error finally ^^!!
it's not 
limitedOneBondedCode =[
["0481911c-f906-40ee-acef-d4ad39c2d767"],
["dc7600e3-a132-42b0-b96b-67ca0cee4aff"]
]

but 
limitedOneBondedCode =[
"0481911c-f906-40ee-acef-d4ad39c2d767",
"dc7600e3-a132-42b0-b96b-67ca0cee4aff"
]

File: o8g/Scripts/actions.py [hunk 3]
```diff
@@ -629,6 +570,8 @@ def loadBasicWeaknesses(group, x = 0, y = 0):
     else:
         notify("{}'s Basic Weakness Deck already loaded.".format(me))
 
+
+
```
Comment (ChristopherS) [inline o8g/Scripts/actions.py:574]: two Linebreaks for some reason

File: o8g/Scripts/bonded.py [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+    for deckcard in deck:
+        for dic in Bonded:
+            if dic["sourceCard"] == deckcard.Name:
+                #Filter Dream Diary with wrong subtitle
+                if deckcard.Name == DreamDiary["name"] and deckcard.Subtitle != DreamDiary["subtitle"]:
+                    continue              
+                listBonded.extend(dic["bondedCode"])
+    #Remove useless multiple bonded card and add only 1 bonded card
+    #Essence of the Dream
+    #Pendant of the Queen
```
Comment (ChristopherS) [inline o8g/Scripts/bonded.py:39]: obsolete comments

File: o8g/Scripts/bonded.py [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+    for idOne in limitedOneBondedCode:
+        if idOne in listBonded:
+            listBonded = filter(lambda card: card != idOne, listBonded)
+            listBonded.append(idOne)
+    return listBonded
```
Comment (ChristopherS) [inline o8g/Scripts/bonded.py:44]: No newline at end of file

**Review by ChristopherS — CHANGES_REQUESTED** [review]
I would just request a bit of clean up on the code. Otherwise it looks good so far, assuming it has been tested.

**Review by ChristopherS — APPROVED** [review]
I'd say this one is good to go.

**Pull request merged**